services/velero/3.4.0/post-install/deployment_update-backupstoragelocation.yaml

```
@@ -32,21 +32,32 @@ subjects:
     name: velero-post-install
     namespace: ${releaseNamespace}
 ---
-apiVersion: batch/v1
-kind: Job
+apiVersion: apps/v1
+kind: Deployment
 metadata:
-  name: velero-post-install
+  name: velero-backup-storage-location-informer
```

```suggestion
  name: velero-backup-storage-location-updater
```
(nit) maybe `-reconciler` or `-fixer` or sth more reflecting of the "update" action ?